src/bidiMapper/domains/context/context.ts

```diff
@@ -82,6 +96,36 @@ export abstract class Context {
     this.#parentId = parent;
     this.#sessionId = sessionId;
     this.cdpClient = cdpClient;
+
+    this.setCDPListeners();
+  }
+
+  setCDPListeners() {
+    this.cdpClient.Page.on(
+      'lifecycleEvent',
+      async (params: Protocol.Page.LifecycleEventEvent) => {
```

Comment:
If there's no `await`, you should not need `async`. Same below.

Reply:
done